frontend/packages/dev-console/src/components/topology/components/nodes/PodSet.tsx

```diff
@@ -51,6 +69,16 @@ const PodSet: React.FC<PodSetProps> = ({ size, data, x = 0, y = 0 }) => {
         outerRadius={podStatusOuterRadius}
         data={completedDeploymentData}
         size={size}
+        subTitle={showPodCount && subTitle}
+        {...showPodCount &&
+          !accessAllowed && {
```

Comment:
Why the check for `!accessAllowed` ?

Reply:
refactor this to be more readable.. this check is for knative revisions https://github.com/openshift/console/pull/3746/commits/a80b87ac4da0bd2e9f6497575695b3ace7425d62